### PR TITLE
add support for status-version file

### PR DESCRIPTION
**Pull request by Sispheor**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ manage_firewall_rules: true
 openvpn_client_register_dns: true
 openvpn_use_crl: false
 openvpn_duplicate_cn: false
+openvpn_status_version: 2
```
Comment (kyl191) [inline defaults/main.yml]: According to https://community.openvpn.net/openvpn/wiki/Openvpn24ManPage, the default is 1, not 2.
Could you change it?

**Comment by Sispheor** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ Role Variables
 | openvpn_use_crl                    | boolean | true , false |                                                | Configure OpenVPN server to honor certificate revocation list.                                                                                                    |
 | openvpn_client_register_dns        | boolean | true , false | true                                           | Add `register-dns` option to client config (Windows only).                                                                                                        |
 | openvpn_duplicate_cn               | boolean | true , false | false                                          | Add `duplicate-cn` option to server config - this allows clients to connect multiple times with the one key.                                                      |
+| openvpn_status_version             | int     | 1, 2, 3      | 2                                              | Define the formatting of the openvpn-status.log file where are listed current client connection                                                                    |
```
Comment (kyl191) [inline README.md]: Got to change the default here as well :P

Comment (Sispheor) [reply README.md]: Forgot this one ;).
Done !

**Pull request merged**